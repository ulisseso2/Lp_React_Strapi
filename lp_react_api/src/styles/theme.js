export const theme = {
  colors: {
    primary: '#457B9D',
    light: '#A8DADC',
    dark: '#1D3557',
    hot: '#E63946',
    white: '#f7fff7',
    black: '#343434',
  },
  fonts: {
    family: {
      main: "'Open Sans', sans-serif",
      title: "'Montserrat', sans-serif",
      body: "'Open Sans', sans-serif",
    },
  },
};
//tema padrão da aplicação usado em vários componentes
