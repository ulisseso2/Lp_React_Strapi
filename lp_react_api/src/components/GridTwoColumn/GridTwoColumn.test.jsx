import { screen } from '@testing-library/react';
import { GridTwoColumn } from '.';
import { renderTheme } from '../../styles/render.theme';

describe('<GridTwoColumn>', () => {
  it('should render with default values', () => {
    renderTheme(<GridTwoColumn>Oi</GridTwoColumn>);
    expect(screen.getByRole('heading')).toBeInTheDocument();
  });
});
