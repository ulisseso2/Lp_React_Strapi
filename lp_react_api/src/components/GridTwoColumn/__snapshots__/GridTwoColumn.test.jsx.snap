// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridTwoColumns> should render Two Columns grid 1`] = `
<div>
  <div
    class="sc-gKXOVf dgAgri"
    id=""
  >
    <div
      class="sc-jSMfEi cRkChI"
    >
      <div
        class="sc-gsnTZi gLKscr"
      >
        <div
          class="sc-dkzDqf hOBXQg"
        >
          <h2
            class="sc-bczRLJ hGDuJC"
          >
            Grid two columns
          </h2>
          <p
            class="sc-iBkjds jknRkk"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Distinctio magnam culpa eveniet doloribus harum? Ipsam, a necessitatibus? Sequi sunt accusantium quod, animi iure a, aliquid dolor ea vel magni dolore?
          </p>
        </div>
        <div
          class="sc-hKMtZM ixKqbH"
        >
          <img
            alt="Grid two columns"
            class="sc-eCYdqJ fsGMuB"
            src="assets/images/js.jpeg"
          />
        </div>
      </div>
    </div>
  </div>
   
</div>
`;
